solutions/cses-1640.mdx

```diff
@@ -82,15 +82,10 @@ $j < i$.
 #include <bits/stdc++.h>
 using namespace std;
 using ll = long long;
-using vi = vector<int>;
 #define pb push_back
-#define rsz resize
-#define all(x) begin(x), end(x)
-#define sz(x) (int)(x).size()
-using pi = pair<int,int>;
 #define f first
 #define s second
-#define mp make_pair
+	
 void setIO(string name = "") { // name is nonempty for USACO file I/O
```

Comment:
this function isn't even used 👀 ??

Reply:
i thought i removed that 👀 (yeah that isn't used)